modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -31,7 +31,7 @@ paths:
                     $ref: '#/components/schemas/Foo'
   /pet:
     servers:
-    - url: 'http://petstore.swagger.io/v2'
+    - url: 'https://petstore.swagger.io:443/v2'
```

Comment:
why is protocol AND port change required? Shouldn't port be automatically deducted from the protocol?

Reply:
Hi @agilob, I'm about to remove this PR, as it is irrelevant after @wing328 comment! :wink:
I'm only waiting for him to answer about the perl bugfix...

Reply:
please file a separate PR for the perl bug fix.

Reply:
I've filed https://github.com/OpenAPITools/openapi-generator/pull/12897 to fix the Perl petstore issue. Not sure if that's the one you referred to.

If not, please file another one.

I've also added perl petstore tests to Travis CI so we no longer rely on running perl tests locally.

Reply:
Hi @wing328,
I've filled #12983 with the bug I found (which is not the one you targeted :wink: ).
I'm actually curious: it the file I fixed part of the Travis CI tests? Because without this fix, I wasn't able to run it at all...